scripts/lib/docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+
+# ensure_docker [<docker-version>]
```

Comment:
meh, there is no handling of an argument for the version?

Reply:
Right. it will be complex to implement for both daemon/cli let's keep it simple for now